pkg/ride/runtime.go

```diff
@@ -255,14 +553,21 @@ func (a rideRecipient) get(prop string) (rideType, error) {
 }
 
 func (a rideRecipient) String() string {
-	r := proto.Recipient(a)
-	return r.String()
+	switch {
+	case a.Address != nil:
+		return rideAddress(*a.Address).String()
+	case a.Alias != nil:
+		return rideAlias(*a.Alias).String()
```

Comment:
What if `a.Address` and `a.Alias` both is not nil? What should be first?

This is `String` method of `proto.Recipient`.
```go
func (r *Recipient) String() string {
	if r.Alias != nil {
		return r.Alias.String()
	}
	return r.Address.String()
}
```
As you can see, this method returns `Alias` string representation first (and, btw, returns in form `alias:<currentScheme>:<alias>`).